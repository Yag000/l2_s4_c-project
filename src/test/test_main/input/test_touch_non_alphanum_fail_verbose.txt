touch this_Does-Bot.wor
