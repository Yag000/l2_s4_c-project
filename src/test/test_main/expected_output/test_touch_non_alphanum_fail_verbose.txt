Parsing file src/test/test_main/input/test_touch_non_alphanum_fail_verbose.txt ...
/$ touch this_Does-Bot.wor
Invalid path of file.
